### PR TITLE
Disabled cppcheck & no comment when no error found

### PR DESCRIPTION
**Pull request by naubryGV**
I only disabled the curl request to post the cppcheck message, we should still be able to see its output on the console

### CODE REVIEW

File: checkall.sh [hunk 1]
```diff
@@ -110,8 +110,10 @@ OUTPUT+="$PAYLOAD_FLAWFINDER"
 OUTPUT+=$'\n```\n' 
 
 PAYLOAD=$(echo '{}' | jq --arg body "$OUTPUT" '.body = $body')
+if [ $FLAWFINDER_LINE_NUMBER -gt 27 ]
+then
```
Comment (cardinotGV) [inline checkall.sh:114]: can we rely on those line numbers?

Comment (naubryGV) [reply checkall.sh:114]: It's a bit of an ugly way, but that's the quickest solution I could think of, but if you have a better way, please let me know cause I hate hardcoding values like that!

Comment (naubryGV) [reply checkall.sh:114]: So far it 's been consistent from what I could gather

Comment (cardinotGV) [reply checkall.sh:114]: no problem, if it's working well, that should be ok for now 👍

**Review by cardinotGV — APPROVED** [review]

**Pull request merged**